src/ReturnBookUi.java

```diff
@@ -25,25 +50,26 @@ public void run() {
                     break;
                 }
```

Comment:
Add a new line in between lines 51 and 52 to make spacing uniform throughout file.